server/admin_management/tests/constants.py

```diff
@@ -36,6 +38,8 @@
     }
 )
 
+TEST_USER_GUID_IDIR = os.environ.get("USER_GUID_IDIR") or ""
```

Comment:
Hi Ian, I think maybe we can discuss a little bit about how we'll use the user_guid of IDIR user when run tests. For now, I just put a placeholder, and it's just empty. But after we add the user verification in backend, either we provide the user_guid through environment variable from GitHub secret, or we skip the IDIR verification when run tests.

Reply:
discussed that we might don't need to do user validation immediately, so will worry about it later.